src/App.tsx

```diff
@@ -1,14 +1,57 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
+function loadTodos(
+  setIsLoading: (isLoading: boolean) => void,
+): Promise<Todo[]> {
+  setIsLoading(true);
+
+  return getTodos()
+    .finally(() => {
+      setIsLoading(false);
+    });
+}
```

Comment:
maybe better move it to the component. All logic with loading will be in one place ) And I'm not sure that you can reuse this function